step-ca/provisioners.mdx

```diff
@@ -1144,9 +1152,6 @@ TPMs also have privacy constraints and a wide range of use case-specific validat
   </div>
 </Alert>
 
-For a description of how a TPM-based attestation flow might work, if implemented,
-see [TPMs + ACME: Better Together](https://smallstep.com/blog/managed-device-attestation/#tpms--acme-better-together)
-
```

Comment:
Is your reason for deleting this because it shows how things work, or just that it's in the wrong place? I think the diagram and text are at a fairly high level, so not too big of an issue?